data/json/itemgroups/Clothing_Gear/clothing.json

```diff
@@ -2270,6 +2270,7 @@
       [ "armor_riot", 10 ],
       [ "swat_armor", 6 ],
       [ "armor_scrapsuit", 12 ],
+	  [ "armor_tiresuit", 12 ],
```

Comment:
You need to format your JSON content. Here's more information on that: https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-contributors#whats-this-about-linting--what-the-heck-is-a-lint-error

Reply:
Thanks, I think this should now be resolved, I have run that file through the formatter and pushed it again

Reply:
Some other files also need to be formatted.

Reply:
Thank you, I have gone through all the files now so they all should be done.